### ISSUE TITLE
Eufy Intergrtion not initializing

### ISSUE THREAD

**Issue by StevenB1701** [fuatakgun/eufy_security#525]
Hi, Ive been getting this error for a few days now. Have tried reloading, re configuring, re starting HA but no luck. May try deleting and re installing if no solution found.

Thanks

![Capture](https://user-images.githubusercontent.com/59145593/201873227-c6a69770-d787-4501-b638-b665c09f4180.PNG)

This error originated from a custom integration.

Logger: custom_components.eufy_security
Source: custom_components/eufy_security/coordinator.py:90
Integration: Eufy Security (documentation, issues)
First occurred: November 14, 2022 at 6:59:39 PM (541 occurrences)
Last logged: 8:41:22 AM

eufy_security - no devices available yet

**Comment by Glennesink** [comment]
Got the same problem, addon log says the following. 
2022-11-15 09:37:29.148  ERROR Response data is missing 
{
  code: 0,
  msg: 'Succeed.',
  data: undefined
}

**Comment by kauthmbt** [comment]
Could you please take a look in the logging protocol of the eufy security add-on?

I am currently experiencing issues as well on a well working environment and maybe we share the same root cause.
My logging is suddenly like this:
```
2022-11-15 10:44:04.480  INFO  Eufy Security server listening on host 0.0.0.0, port 3000 
2022-11-15 10:47:18.181  INFO  No houses found. 
2022-11-15 10:47:18.186  INFO  No stations found. 
2022-11-15 10:47:18.190  INFO  No devices found. 
2022-11-15 10:47:21.957  INFO  No houses found. 
2022-11-15 10:47:21.960  INFO  No stations found. 
2022-11-15 10:47:21.962  INFO  No devices found. 
```

**Comment by SupaHotMojo** [comment]
Indeed, I did also fix this by going to:
HA Add-ons ->  Eufy Security add-on -> configuration tab.
Where you also be able to enter your  email + pass and event duration.
Last one will be TRUSTED_DEVICE_NAME.
Just enter some random name like smackles or just some jibberish.

**Comment by StevenB1701** [comment]
Hi Fuatakgun/Eufy_security

I've just tried this and so far it has worked!

Thanks 🙂

**Comment by slyicep** [comment]
Adding a TURSTED_DEVICE_NAME resolved it for me as well.  I'm not sure when it stopped, but I'm thinking one of the last Home Assistant updates I completed recently because it started in the last few days.

**Comment by Glennesink** [comment]
Worked!!!

**Comment by advicepyro** [comment]
> Hi, Came across this as I was running into a similar error. Please take a look here: [bropat/eufy-security-client#242](https://github.com/bropat/eufy-security-client/issues/242)
> 
> Long story short: set the value of **TRUSTED_DEVICE_NAME** in the configuration of Eufy Security Add-on to any value that you like (e.g. HomeAssistant). Helped me out to fix this.

Thanks, that worked! Can we pin this issue for other users wondering why their integration broke?

**Comment by golles** [comment]
Thanks for sharing, this was not trivial to find out!

**Comment by buzurk26** [comment]
Same happening here.

**Comment by kvanhoorn** [comment]
I have the captcha verification at boot. Tried some different TRUSTED names as well.
Home Assistant will nog popup the captcha at boot, only if has been succesfully receiving devices before.
Can I interactively answer the captcha?

I see that I get this message:

` type: 'event',
  event: {
    source: 'driver',
    event: 'captcha request',
    captchaId: '11111-1-1-1-1-1111',
    captcha: 'data:image/png;base64,yadayada='`

I set up a connection manually by using:

`./node_modules/.bin/ts-node src/bin/client.ts --host x.x.x.x --port 3000 --verbose`

What do I reply when this message comes by? I tried:

`driver.set_captcha 'aAbB' '11111-1-1-1-1-1111'`

But it's not working, please help.

**Comment by exeljb** [comment]
Oddly this just started happening to me yesterday after a power outage. Anyway, setting `TRUSTED_DEVICE_NAME` to something ( I entered HomeAssistant) other than the field being blank and reloading the integration instantly resolved the retrying setup issue for me as well.

I mistakenly looked in HA at a couple other places for this like a dope, so avoid being like me and just go to Settings > Add-ons > Eufy Security Add-on. Then the Configuration tab, in the Options section is where the 'TRUSTED_DEVICE_NAME' field is.  Enter anything in there, and hit Save.

After that, I reloaded the integration by going to Settings > Devices & Services.  Then the Integrations tab, click the 3 vertical dots for the Eufy Security integration.  In the pop up menu, hit Reload and it should bring your Eufy stuff back from unkownland.

Thank you for the help
